pom.xml

```diff
@@ -604,7 +604,7 @@
            <groupId>org.jolokia</groupId>
            <artifactId>jolokia-war</artifactId>
            <type>war</type>
-           <version>1.3.5</version>
+           <version>1.3.7</version>
```

Comment:
Seems like we should move the &lt;jolokia-version> property from artemis-hawtio/pom.xml into here and then use that in both places so it's easier to keep in sync.

Reply:
+1

Reply:
that's an easy change

Reply:
I see this got merged in overnight (uk) , I’ll raise a new pr to address this, agreed best to have one place for versions